main.py

```diff
@@ -0,0 +1,253 @@
+import pygame
+import sys
+
+clock=pygame.time.Clock()
+
+from pygame.locals import *
+
+import random
+
+pygame.mixer.pre_init(44100, -16, 2, 2048)
+pygame.init()
+pygame.mixer.set_num_channels(64)
+
+pygame.display.set_caption('    World Runner')
+
+WINDOW_SIZE= (600,400)
+
+screen =pygame.display.set_mode(WINDOW_SIZE)
+
+display = pygame.Surface((300,200))
+
+player_image = (pygame.image.load('player_animations/idle/idle1.jpg').convert_alpha())
+
+grass_image = pygame.image.load('map_files/grass.jpg').convert_alpha()
+grass_image= pygame.transform.scale(grass_image,(16,16))
+TILE_SIZE = grass_image.get_width()
+lava_image=pygame.image.load('map_files/lava.png').convert_alpha()
+lava_image= pygame.transform.scale(lava_image,(16,16))
+plant_image=pygame.image.load('map_files/plant.png').convert_alpha()
+plant_image= pygame.transform.scale(plant_image,(16,16))
+
+tile_index={1:grass_image,2:lava_image,3:plant_image}
+
+chunk_size=8
+
+def generate_chunk(x,y):
+  chunk_data=[]
+  for y_pos in range(chunk_size):
+    for x_pos in range(chunk_size):
+      target_x=x*chunk_size+x_pos
+      target_y=y*chunk_size+y_pos
+      tile_type=0
+      if target_y >10:
+        tile_type=2
+      elif target_y==10:
+        tile_type=1
+        if random.randint(1,5)==2:
+          tile_type=0
+      elif target_y==9:
+        if random.randint(1,5)==1:
+          tile_type=3
+      if tile_type!=0:
+        chunk_data.append([[target_x,target_y],tile_type])
+  return chunk_data
+   
+game_map={}
```

Comment:
what is this dict used for? I would recommend putting all your functions together and your main code afterward.